src/plugins/plot/inspector/PlotOptionsBrowse.vue

```diff
@@ -73,7 +73,7 @@
                 </div>
             </li>
             <li
-                v-if="!yAxis.autoscale && yAxis.rangeMin"
+                v-if="!yAxis.autoscale && yAxis.rangeMin !== ''"
```

Comment:
Same, we can remove the `!== ''` here

Reply:
We want to allow rangeMax and rangeMin 0 values. This is why I've specified ''